### PR TITLE
EDU-38 Add scripts for updating S3 objects metadata

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes #38

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,34 @@
 # elmu-tools
 
-Tools for elmu development and maintenance
+Tools for elmu development and maintenance.
```
Comment (laurentiu-ilici) [inline README.md:3]: Nice docu!

File: s3-helper.js [hunk 2]
```diff
@@ -51,8 +51,34 @@ function copyObject(s3, sourceBucketName, sourceKey, destinationBucketName) {
...
+    Metadata: metadata,
+    ContentType: contentType
+  };
+
+  return new Promise((resolve, reject) => {
+    s3.putObject(params, (err, s3Data) => err ? reject(err) : resolve(s3Data));
+  });
+}
+
+function getObjectHead(s3, bucketName, key) {
```
Comment (laurentiu-ilici) [inline s3-helper.js:67]: Why not use ({ s3, bucketName, ...}) here as well.

**Review by laurentiu-ilici — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -28,8 +28,9 @@
     "eslint-plugin-import": "~2.25.2",
     "execa": "~2.0.4",
     "faker": "~5.5.3",
+    "mime": "~3.0.0",
```
Comment (ahelmberger) [inline package.json:31]: Nice :-)

File: README.md (hunk 1 above)
Comment (adrianaluchian) [reply README.md:3]: Danke! 🙏

File: s3-helper.js (hunk 2 above)
Comment (adrianaluchian) [reply s3-helper.js]: Overlooked it cause it grew in arguments. Updated, thanks!

File: README.md [hunk 4]
```diff
@@ -1,6 +1,34 @@
...
+  Anonymizes all sensitive user information, replacing it with mock data.
+2. `dump-stag`
+  Creates a mongodump of the staging MongoDb database locally in `./tmp/dump/stag-elmu-web`
+3. `dump-prod`
+  Creates a mongodump of the production MongoDb database locally in `./tmp/dump/prod-elmu-web`
+4. `update-metadata-stag`
+  Updates the metadata of each S3 object in a specified bucket by:
+  - ensuring a `x-amz-meta-createdon` value in the metadata, set to the value of `LastModified`
+  - ensuring a correct `Content-Type` value in the metadata, according to the type of the data saved in the object
+## Environment variables
```
Comment (ahelmberger) [inline README.md:18]: Where's `update-metadata-prod` ?

File: package.json (hunk 3 above)
Comment (adrianaluchian) [reply package.json:31]: Wella, it's the same fix you did in `elmu-web` basically :)

File: update-metadata-prod [hunk 5]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
```
Comment (adrianaluchian) [inline update-metadata-prod:1]: @ahelmberger It's'a here

**Review by laurentiu-ilici — APPROVED** [review]

File: update-metadata-prod (hunk 5 above)
Comment (ahelmberger) [reply update-metadata-prod:1]: I meant in the nice docu ;-)

Comment (adrianaluchian) [reply update-metadata-prod:1]: Ge-added, sorry 😁

Comment (ahelmberger) [reply update-metadata-prod:1]: Ge-thanked!

Comment (laurentiu-ilici) [reply update-metadata-prod:1]: Ge-thanks!

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**